README.md

```diff
@@ -50,16 +50,6 @@ We must carefully consider any additional client-side libraries that we include
 
 For any improvements on the website, we should continuously measure its impact on the [Core Web Vitals](https://web.dev/vitals/). If proven to hurt the metrics, we should revert the changes and find another way to implement them without degrading the performance.
 
-### Testing Guidelines
-
-In general, these are three guidelines that we need to pay attention to:
-
-1. [Query priorities](https://testing-library.com/docs/queries/about#priority)
-2. [Testing appearance and disappearance](https://testing-library.com/docs/guide-disappearance/)
-3. [Opting in for interactions instead of events](https://testing-library.com/docs/guide-events/#interactions-vs-events)
-
-Please ensure that all queries and assertions in your test files comply with those testing guidelines.
-
```

Comment:
Your PR keeps undoing the changes from the previous PRs. I'm wondering how you did it.

Please make sure that there's no changes other than your own changes reflected in this PR. Thanks!